sentry-android-core/src/test/java/io/sentry/android/core/ActivityLifecycleIntegrationTest.kt

```diff
@@ -20,7 +20,7 @@ import io.sentry.SentryLevel
 import io.sentry.SentryOptions
 import io.sentry.SentryTracer
 import io.sentry.SpanStatus
-import io.sentry.TraceState
+import io.sentry.TraceContext
```

Comment:
Renamed `TraceState` to `TraceContext` to avoid confusion as it has nothing to do with the `tracestate` header and also to align naming with relay and other SDKs.